README.md

```diff
@@ -408,7 +408,7 @@ causing the browser to navigate to a new page should:
 class LoginPage < AePageObjects::Document
   path :new_user_session
 
-  def login!(username, password)
+  def login_as!(username, password)
```

Comment:
Thanks for the PR!
could you change all the usages or definitions for `login!` method in this doc?

Reply:
👍 will do